src/service-override/views.ts

```diff
@@ -739,5 +739,9 @@ export {
   SidebarPart,
   ActivitybarPart,
   PanelPart,
-  Parts
+  Parts,
+
+  IActivityService,
+  IViewDescriptorService,
+  IPaneCompositePartService
```

Comment:
We prefer exporting service identifiers from `services.ts` to be able to use a custom implementation without pulling vscode implementations (and not use it)

Reply:
Sounds good, I'll put it there!

Reply:
Noticed just now that they are in `services` 🤦 . Looks like my editor couldn't find them. I'll close this PR.